components/apps.js

```diff
@@ -173,8 +299,8 @@ class SteamUserApps extends SteamUserAppAuth {
 			inclTokens = false;
 		}
 
-		// This one actually can take a while, so allow it to go as long as 60 minutes
-		return StdLib.Promises.timeoutCallbackPromise(3600000, ['apps', 'packages', 'unknownApps', 'unknownPackages'], callback, (resolve, reject) => {
+		// This one actually can take a while, so allow it to go as long as 120 minutes
+		return StdLib.Promises.timeoutCallbackPromise(7200000, ['apps', 'packages', 'unknownApps', 'unknownPackages'], callback, async (resolve, reject) => {
```

Comment:
Increased to 2h, because one of the top game owners on steam is trying to use `node-steam-user` but has issues getting passed this point.

Reply:
> Increased to 2h, because one of the top game owners on steam is trying to use `node-steam-user` but has issues getting passed this point.

Btw, I can confirm with all my changes, this person was able to finally get his ownership cached. 🎉🎉